shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -195,15 +195,9 @@
  * screen until the Android application is initialized.
  *
  * <p>Flutter also requires initialization time. To specify a splash screen for Flutter
- * initialization, subclass {@code FlutterActivity} and override {@link #provideSplashScreen()}. See
- * {@link SplashScreen} for details on implementing a splash screen.
- *
- * <p>Flutter ships with a splash screen that automatically displays the exact same {@code
- * windowBackground} as the launch theme discussed previously. To use that splash screen, include
- * the following metadata in AndroidManifest.xml for this {@code FlutterActivity}:
- *
- * <p>{@code <meta-data android:name="io.flutter.embedding.android.SplashScreenDrawable"
- * android:resource="@drawable/launch_background" /> }
+ * initialization, DO NOT subclass {@code FlutterActivity} and override {@link
+ * #provideSplashScreen()}. Instead, use the instructions above to specify a launch screen that
```

Comment:
Is there a way to link to it?

Reply:
If not, please specify where above as there are multiple sections and this can be confusing.

Reply:
I rewrote the comment entirely to catch something I missed and to avoid this strange referencing.